components/molecule/autosuggest/README.md

```diff
@@ -0,0 +1,88 @@
+# MoleculeAutosuggest
+
+`MoleculeAutosuggest` is an input that will display a list of suggestions while values are entered
+It allows Single and Multiple Selection
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-molecule-autosuggest --save
+```
+
+## Usage
+
+```js
+import MoleculeAutosuggest, {
+  MoleculeAutosuggestDropdownListSizes,
+  MoleculeAutosuggestOption
```

Comment:
looks like you'r not exporting this anymore

Reply:
fixed!